### ISSUE TITLE
elastic transforms add weird artifacts on images

### ISSUE THREAD

**Issue by Jimmy2027** [MonsieurWave/PerfusionCT-Net#17]
Using the following configuration for the augmentation values results in some images that have artifacts.

**Augmentation values**
```
"rotate": 0,
"scale_val": [1,1],
"max_deform": [10, 10, 10],
"intensity": [1,1],
"random_flip_prob": 0,
"random_affine_prob": 0,
"random_elastic_prob": 1,
"scale_size": [96,128,128,1],
"random_noise_prob": 0
```

**Examples of resulting images**
![image](https://user-images.githubusercontent.com/43926224/84012904-8cb40880-a978-11ea-9b27-6f30a8a602f2.png)
![image](https://user-images.githubusercontent.com/43926224/84012952-96d60700-a978-11ea-92dc-998846c1b0c6.png)
![image](https://user-images.githubusercontent.com/43926224/84012984-9fc6d880-a978-11ea-84f4-7ed941a842e7.png)

**Examples of not transformed images**
![image](https://user-images.githubusercontent.com/43926224/84013080-ba00b680-a978-11ea-8b32-197c72d8b563.png)

**Comment by Jimmy2027** [comment]
The same issue happens when only scaling the image with config:

```
"rotate": 0,
"scale_val": [0.7,1.2],
"max_deform": [0, 0, 0],
"intensity": [1,1],
"random_flip_prob": 0,
"random_affine_prob": 1,
"random_elastic_prob": 0,
"scale_size": [96,128,128,1],
"random_noise_prob": 0
```

![image](https://user-images.githubusercontent.com/43926224/84033212-cd237e80-a998-11ea-88cd-fc219c288c73.png)
![image](https://user-images.githubusercontent.com/43926224/84033228-d280c900-a998-11ea-81cf-6529f1f4579f.png)
![image](https://user-images.githubusercontent.com/43926224/84033240-d7de1380-a998-11ea-80af-0f97a6a9d663.png)

**Comment by JulianKlug** [comment]
Might be fixed by #e3d8d9dc4010282b36d22fb80a12b4afa03d9a97
